src/admin/class.component.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect, ReactDOM } from 'react';
+import Config from '../config/app.local.conf';
+import { Table, Divider, Tag } from 'antd';
+import { isEmpty } from 'lodash';
+import AddClassModal from './modalSubmit.component';
+import Axios from 'axios';
+
+
+function Class() {
+  const [classes, setClasses] = useState([]);
+  const { Column } = Table;
+
+  useEffect(() => {
+    if (isEmpty(classes)) {
+      loadData();
+    }
+  })
+
+  const loadData = () => {
+    //alert('update');
+    fetch(Config.websiteServiceUrl + "class")
+      .then(res => res.json())
```

Comment:
You also want to handle failure paths, you can do that with this method:
`  .catch(function (error) {
    // handle error
    console.log(error);
  })`

Reply:
you'll want to do the same on the delete fetch as well

Reply:
In the case of errors, you'll also want to display a notification to the user that there was an error:

https://ant.design/components/notification/